app/experimenter/experiments/constants/nimbus.py

```diff
@@ -379,308 +379,213 @@ class NimbusTargetingConfig:
-    "(currentDate|date - profileAgeCreated|date) / 86400000 >= 28",
+    targeting=f"{TARGETING_STICKY} || {TARGETING_PROFILE28DAYS} && "
+    "userMonthlyActivity|length >= 1 && userMonthlyActivity|length <= 6",
     desktop_telemetry="",
     application_choice_names=(Application.DESKTOP.name,),
 )
 
-TARGETING_INFREQUENT_USER_NEED_PIN = NimbusTargetingConfig(
+INFREQUENT_USER_NEED_PIN = NimbusTargetingConfig(
     name="Infrequent user (need pin)",
     slug="infrequent_user_need_pin",
     description="Between 1 and 6 days of activity in the past 28 days needing pin",
-    targeting="{infrequent_user} && doesAppNeedPin || {sticky}".format(
-        infrequent_user=TARGETING_INFREQUENT_USER_URIS.targeting,
-        sticky=TARGETING_STICKY,
-    ),
+    targeting=f"{INFREQUENT_USER_URIS.targeting} && doesAppNeedPin",
     desktop_telemetry="",
     application_choice_names=(Application.DESKTOP.name,),
 )
 
-TARGETING_INFREQUENT_USER_NEED_DEFAULT = NimbusTargetingConfig(
+INFREQUENT_USER_NEED_DEFAULT = NimbusTargetingConfig(
     name="Infrequent user (need default)",
     slug="infrequent_user_need_default",
     description="Between 1 and 6 days of activity in the past 28 days needing default",
-    targeting="{infrequent_user} && !isDefaultBrowser || {sticky}".format(
-        infrequent_user=TARGETING_INFREQUENT_USER_URIS.targeting,
-        sticky=TARGETING_STICKY,
-    ),
+    targeting=f"{INFREQUENT_USER_URIS.targeting} && {TARGETING_NEED_DEFAULT}",
     desktop_telemetry="",
     application_choice_names=(Application.DESKTOP.name,),
 )
 
-TARGETING_INFREQUENT_USER_NEED_DEFAULT_HAS_PIN = NimbusTargetingConfig(
+INFREQUENT_USER_NEED_DEFAULT_HAS_PIN = NimbusTargetingConfig(
     name="Infrequent user (need default, has pin)",
     slug="infrequent_user_need_default_has_pin",
     description="Between 1 & 6 days activity in past 28 days need default w/ pin",
-    targeting=("{infrequent_user} && {need_default} && {has_pin} || {sticky}").format(
-        infrequent_user=TARGETING_INFREQUENT_USER_URIS.targeting,
-        need_default=TARGETING_NEED_DEFAULT,
-        has_pin=TARGETING_HAS_PIN,
-        sticky=TARGETING_STICKY,
-    ),
+    targeting=f"{INFREQUENT_USER_NEED_DEFAULT.targeting} && {TARGETING_HAS_PIN}",
     desktop_telemetry="",
     application_choice_names=(Application.DESKTOP.name,),
 )
 
-TARGETING_INFREQUENT_USER_HAS_DEFAULT_NEED_PIN = NimbusTargetingConfig(
+INFREQUENT_USER_HAS_DEFAULT_NEED_PIN = NimbusTargetingConfig(
     name="Infrequent user (has default, need pin)",
     slug="infrequent_user_has_default_need_pin",
     description="Between 1 & 6 days activity in past 28 days w/ default need pin",
-    targeting=(
-        "{infrequent_user} && isDefaultBrowser && doesAppNeedPin || {sticky}"
-    ).format(
-        infrequent_user=TARGETING_INFREQUENT_USER_URIS.targeting,
-        sticky=TARGETING_STICKY,
-    ),
+    targeting=f"{INFREQUENT_USER_NEED_PIN.targeting} && isDefaultBrowser",
```

Comment:
Not sure if I went too far with this latest commit? Here I reused `INFREQUENT_USER_NEED_PIN` instead of the original base `INFREQUENT_USER_URIS` as it already has one of the additional conditions. So along with the shorter variable name and now only 1 extra condition, this can fit on a single line now.